src/components/Person/Person.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Person.scss';
+
+export const Person = ({ name, age, sex, isMarried, partnerName }) => {
+  const partner = sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <div className="Person person__container">
+      <section className="Person Person__section">
+        <h2 className="Person__name">
+          {`My name is ${name}`}
+        </h2>
+        <p className="Person__age">
+          {age
+            ? (
+              `I am ${age}`)
+            : (
+              null
+            )}
+        </p>
```

Comment:
What if we don't get an age? Will we render empty p? Could we don't render p if we don't have age value?